standard/region.lua

```diff
@@ -0,0 +1,52 @@
+local region = {}
+local Class = require('Module:Class')
+local Flag = require('Module:Flags')
+local String = require('Module:String')
+local Lua = require('Module:Lua')
+local regionData = mw.loadData('Module:Region/Data')
+local countryToRegionData = Lua.loadDataIfExists('Module:Region/CountryData', {})
+
+function region.run(args)
+	args = args or {}
+	local regionEntry = args.region
+	local onlyRegion = args.onlyRegion == 'true'
```

Comment:
```suggestion
	local shouldOnlyReturnRegion = args.onlyRegion == 'true'
```

Reply:
This would benefit from an even better name though, because returning the "region" from a module called "Region" is a bit vague.

Reply:
used `shouldOnlyReturnRegionName`